Assets/Scripts/MovableCamera.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using UnityEngine;
+
+namespace DigThemGraves
+{
+    [RequireComponent(typeof(Camera))]
+    public class MovableCamera : MonoBehaviour
+    {
+        [SerializeField]
+        private float acceleration;
+        [SerializeField]
+        private bool inverseSteering;
+
+        private UserSwipe swipeGateway;
+        private Camera targetCamera;
+        void Awake()
+        {
+            swipeGateway = UserSwipe.Create();
+            targetCamera = GetComponent<Camera>();
+        }
+
+        void LateUpdate()
+        {
+            var swipeData = swipeGateway.Swipe;
+            if (!swipeData.Swiped)
+            {
+                return;
+            }
+
+            var direction = swipeData.Direction;
```

Comment:
Nie podoba mi się używanie varów. Nie wiadomo jakiego typu są to zmienne co utrudnia czytanie. (Masz vary jeszcze w innych skryptach tu)